libs/blocks/hero-marquee/hero-marquee.css

```diff
@@ -17,6 +17,7 @@
   align-items: center;
 }
 
+.hero-marquee.no-min-height { min-height: unset; }
```

Comment:
FYI, I added this from a reverted [PR](https://github.com/adobecom/milo/pull/3002/files#diff-d65b338a88b11eb90de68f7e4682c899d28437788d7e556c20b731de1b311432) right before code freeze. This has zero-impact, however GWP is urgently waiting for this. 
The other piece of that pr addressing the grid w/ no-bg will be a follow up PR.